### DISCUSSION TITLE
How do we redirect to same screen with update url on mobile app ?

### DISCUSSION THREAD

**Discussion by rpatidar** [nandorojo/solito#258] [General]
i am using `Link` as of now where it does not work when redirecting to the same screen. As of now i have a url that looks like `https//someweb/myscreen/id1` when i change this to `https//someweb/myscreen/id2` this works on the web but not on the mobile app. 

as a workaround, i created a pressable event and manually handling the routing, this is poor hack but working for me as of now. 

```
const onItemClick = (path) => {
  if (Platform.OS == 'android' || Platform.OS == 'ios') {
    push('/')
  }
  push(path)
}
//path will be like myscreen/id2
```

any suggestion ? how to do it right way

**Comment by nandorojo** [comment]
It’s not clear what you’re trying  to do. Maybe you want to use `useParam` instead of navigate to a new page?
